BuilderPattern.java

```diff
@@ -0,0 +1,54 @@
+package Ex4;
+public class BuilderPattern{
+	public static void main(String[] args) {
+        Car myCar=new Car.BuildCar().setCompany("Honda").setModel("Civic").setEngine("Petrol").setTire("MRF").build();
+        System.out.println(myCar);
+	}
+}
+class Car{
+    class CarBuilder{
+    String tire, engine, model, company;
+    public CarBuilder(String company,String model,String engine,String tire) {
```

Comment:
Whether public is needed here

Reply:
Not needed bro...I created the constructor by Auto generated stub public comes by default bro